src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -213,27 +213,39 @@ void LayoutButton(UIButton platformButton, Button button, Rect size)
 				var sharedSpacing = spacing / 2;
 
 				// These are just used to shift the image and title to center
-				// Which makes the later math easier to follow
-				imageInsets.Left += titleWidth / 2;
-				imageInsets.Right -= titleWidth / 2;
-				titleInsets.Left -= imageWidth / 2;
-				titleInsets.Right += imageWidth / 2;
+				// Which makes the later math easier to follow				
```

Comment:
These tests are failing based on these changes:
<img width="338" alt="image" src="https://github.com/user-attachments/assets/575084bf-e56f-4218-a5c5-0d5db2c288d9">
Could you review it?

Reply:
@jsuarezruiz , As mentioned in the comment, this fix causes the image to shift by 1 pixel, leading to some test cases failing on the iOS platform. To resolve this, I replaced the images with new ones generated by CI. After running the tests locally, all test cases passed successfully.